.github/workflows/ci.yml

```diff
@@ -95,10 +95,10 @@ jobs:
             timeout: 60
             allow_failure: false
           - name: linux-gcc-tests-asan
-            os: ubuntu-22.04
+            os: ubuntu-24.04
```

Comment:
i think the main reason we had 22 was due the debug tests.

Reply:
@kazarmy

Reply:
it's not yet known whether the debug tests are affected but i guess somebody will have to fix them (or they are ignored for now in the asan builds until all linux builds are at ubuntu-24.04)

Reply:
For now I just want to see those new warnings, like I already saw for capstone code.